x/ccv/consumer/keeper/relay.go

```diff
@@ -83,23 +82,26 @@ func (k Keeper) UnbondMaturePackets(ctx sdk.Context) error {
 	if !ok {
 		return sdkerrors.Wrap(channeltypes.ErrChannelNotFound, "provider channel not set")
 	}
-	channelCap, ok := k.scopedKeeper.GetCapability(ctx, host.ChannelCapabilityPath(types.PortID, channelID))
-	if !ok {
-		return sdkerrors.Wrap(channeltypes.ErrChannelCapabilityNotFound, "module does not own channel capability")
-	}
 
 	for maturityIterator.Valid() {
-		sequence := types.GetSequenceFromPacketMaturityTimeKey(maturityIterator.Key())
+		vscId := types.GetIdFromPacketMaturityTimeKey(maturityIterator.Key())
 		if currentTime > binary.BigEndian.Uint64(maturityIterator.Value()) {
```

Comment:
The inequality doesn't match the spec. It won't matter in practice but will break automated tests

https://github.com/cosmos/ibc/blob/5ea902172b20d53e209f8c1fda9b11d005d9c110/spec/app/ics-028-cross-chain-validation/methods.md?plain=1#L1602-L1621